src/index.ts

```diff
@@ -1,15 +1,102 @@
+import { DNSoverHTTPS } from "dohdec";
 import { getLogger } from "./util/logger";
 
-const { trace } = getLogger("sayHello");
+const { trace } = getLogger("index");
+
+const DoHResolver = new DNSoverHTTPS();
+
+type RecordTypes = "openatts";
+
+type BlockchainNetwork = "ethereum";
+
+type EthereumAddress = string;
+
+enum EthereumNetworkIds {
+  homestead = 1,
+  ropsten = 3
+}
+interface OpenAttestationsDNSTextRecord {
+  type: RecordTypes;
+  net: BlockchainNetwork;
+  netId: EthereumNetworkIds;
+  address: EthereumAddress;
+  dnssec: boolean;
+}
+
+/**
+ * Returns true for strings that are openattestation records
+ * @param txtDataString e.g: '"openatts net=ethereum netId=3 address=0x0c9d5E6C766030cc6f0f49951D275Ad0701F81EC"'
+ */
+const isOpenAttestationsRecord = (txtDataString: string) => {
+  return txtDataString.startsWith('"openatts');
```

Comment:
Its better if we strip down quotes and start-end white space before checking this. There may be some cases where user may put a white space by mistake.

Reply:
oh yea good idea

Reply:
done